zips/zip-0253.md

```diff
@@ -0,0 +1,80 @@
+
+    ZIP: 253
+    Title: Deployment of the NU6 Network Upgrade
+    Owners: Arya <arya@zfnd.org>
+    Status: Draft
+    Category: Consensus / Network
+    Created: 2024-07-17
+    License: MIT
+    Discussions-To: <https://github.com/zcash/zips/issues/806>
+
+# Terminology
+
+The key word "MUST" in this document are to be interpreted as described in
+BCP 14 [^BCP14] when, and only when, they appear in all capitals.
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [^zip-200].
+
+The terms "Testnet" and "Mainnet" are to be interpreted as described in
+section 3.12 of the Zcash Protocol Specification  [^protocol-networks].
+
+# Abstract
+
+This proposal defines the deployment of the NU6 network upgrade.
+
+# Specification
+
+## NU6 deployment
+
+<!-- TODO: Update these references once ZIP numbers are assigned to the draft ZIPs -->
+
+The primary sources of information about NU6 consensus protocol changes are:
+
+* The Zcash Protocol Specification [^protocol].
+* Network Upgrade Mechanism [^zip-200].
+* Lockbox Funding Streams [^lockbox-funding-streams].
+* Block Reward Allocation for Non-Direct Development Funding [^block-reward-allocation-for-non-direct-dev-fund].
+* Blocks should balance exactly [^coinbase-should-balance-exactly].
+
+The network handshake and peer management mechanisms defined in ZIP 201 [^zip-201] also apply to this upgrade.
+
+The following network upgrade constants [^zip-200] are defined for the NU6 upgrade:
+
+CONSENSUS_BRANCH_ID
+: `0xC8E71055`
+
+ACTIVATION_HEIGHT (NU6)
+: Testnet: `2942000`
+: Mainnet: `2726400`
```

Comment:
We do not give heights in the ZIP until the implementations are ready to deploy their respective releases for testnet and mainnet (even when we "know" what an activation height is likely to be), because once the tuple of `(consensus_branch_id, height)` is specified _anywhere_, any change to the implementations forces us to change the consensus branch ID per ZIP 200.
```suggestion
: Testnet: TBD
: Mainnet: TBD
```